garak/analyze/deep_analysis/analytics.py

```diff
@@ -0,0 +1,264 @@
+        "misleading.FalseAssertion50",
+        "snowball.GraphConnectivity",
+        "snowball.GraphConnectivityMini",
+    ],
+    "training_data": [
+        "grandma.Win10",
+        "grandma.Win11",
+        "leakreplay.GuardianCloze",
+        "leakreplay.LiteratureCloze",
+        "leakreplay.LiteratureCloze80",
+        "leakreplay.NYTCloze",
+        "leakreplay.PotterCloze",
+        "divergence.Repeat",
+        "divergence.RepeatExtended",
+    ],
+    "harm": [
+        "atkgen.Tox",
+        "continuation.ContinueSlursReclaimedSlurs",
+        "continuation.ContinueSlursReclaimedSlursMini",
+        "lmrc.Bullying",
+        "lmrc.Deadnaming",
+        "lmrc.QuackMedicine",
+        "lmrc.SexualContent",
+        "lmrc.Sexualisation",
+        "lmrc.SlurUsage",
+        "topic.WordnetControversial",
+    ],
+    "bypass": [
+        "encoding.InjectAscii85",
+        "encoding.InjectBase16",
+        "encoding.InjectBase2048",
+        "encoding.InjectBase32",
+        "encoding.InjectBraille",
+        "encoding.InjectEcoji",
+        "encoding.InjectHex",
+        "encoding.InjectMime",
+        "encoding.InjectMorse",
+        "encoding.InjectNato",
+        "encoding.InjectROT13",
+        "encoding.InjectZalgo",
+    ],
+}
+
+TIER_2_PROBES = list(set().union(TIER_2_PROBE_GROUPS.values()))
+
+CALIBRATION_DATA = data_path / "calibration" / "calibration.json"
+
+JAILBREAK_FEEDBACK_LOCATION = data_path / "deep_analysis" / "jailbreak.json"
+
+
+@lru_cache
+def load_calibration_data(calibration_filename=CALIBRATION_DATA):
+    with open(calibration_filename, "r", encoding="utf-8") as f:
+        data = json.loads(f.read().strip())
+
+    probe_data = {key.split("/")[0]: value for key, value in data.items()}
+    aggregate_scores = dict()
+    for k, v in TIER_1_PROBE_GROUPS.items():
+        scores = [probe_data[probe]["mu"] for probe in v if probe in probe_data.keys()]
+        minus_sigma_scores = [
+            probe_data[probe]["mu"] - probe_data[probe]["sigma"]
+            for probe in v
+            if probe in probe_data.keys()
+        ]
+        plus_sigma_scores = [
+            probe_data[probe]["mu"] + probe_data[probe]["sigma"]
+            for probe in v
+            if probe in probe_data.keys()
+        ]
+        avg_score = float(np.average(scores)) if scores else 0.0
+        low_score = float(np.average(minus_sigma_scores)) if minus_sigma_scores else 0.0
+        high_score = float(np.average(plus_sigma_scores)) if plus_sigma_scores else 0.0
+        aggregate_scores[k] = {
+            "average": avg_score,
+            "low": low_score,
+            "high": high_score,
+        }
+    return data
+
+
+def load_feedback(feedback_type: str) -> dict:
+    match feedback_type:
+        case "jailbreak":
+            with open(JAILBREAK_FEEDBACK_LOCATION, "r", encoding="utf-8") as f:
+                feedback = json.loads(f.read().strip())
+
+        case _:
+            feedback = dict()
+    return feedback
```

Comment:
can we use feedback files named according to a group and just skip if file not found, allowing us to drop the case statements?